scrapers/mt/actions.py

```diff
@@ -2,358 +2,237 @@
-        "Reconsidered Previous Act; Remains in 3rd Read Gov Amend Process",
-        "Reconsidered Previous Action; Placed on 2nd Reading",
-        "Reconsidered Previous Action; Remains in 2nd Reading Process",
-        "Reconsidered Previous Action; Remains in 3rd Reading Process",
-        "Referred to Committee",
-        "Rereferred to Committee",
-        "Resolution Adopted",
-        "Resolution Failed",
-        "Returned from Enrolling",
-        "Returned to House",
-        "Returned to House Concurred in Governor's Proposed Amendments",
-        "Returned to House Not Concurred in Governor's Proposed Amendments",
-        "Returned to House with Amendments",
-        "Returned to Senate",
-        "Returned to Senate Concurred in Governor's Proposed Amendments",
-        "Returned to Senate Not Concurred in Governor's Proposed Amendments",
-        "Returned to Senate with Amendments",
-        "Returned with Governor's Line-item Veto",
-        "Returned with Governor's Proposed Amendments",
-        "Revised Fiscal Note Printed",
-        "Revised Fiscal Note Received",
-        "Revised Fiscal Note Requested",
-        "Revised Fiscal Note Signed",
-        "Rules Suspended to Accept Late Return of Amended Bill",
-        "Rules Suspended to Accept Late Transmittal of Bill",
-        "Scheduled for 2nd Reading",
-        "Scheduled for 3rd Reading",
-        "Scheduled for Consideration under Special Orders",
-        "Scheduled for Executive Action",
-        "Segregated from Committee of the Whole Report",
-        "Sent to Enrolling",
-        "Signed by Governor",
-        "Signed by President",
-        "Signed by Speaker",
-        "Special Note",
-        "Sponsor List Modified",
-        "Sponsor Rebuttal to Fiscal Note Printed",
-        "Sponsor Rebuttal to Fiscal Note Received",
-        "Sponsor Rebuttal to Fiscal Note Requested",
-        "Sponsor Rebuttal to Fiscal Note Signed",
-        "Sponsors Engrossed",
-        "Tabled in Committee",
-        "Taken from 2nd Reading; Rereferred to Committee",
-        "Taken from 3rd Reading; Placed on 2nd Reading",
-        "Taken from Committee; Placed on 2nd Reading",
-        "Taken from Table in Committee",
-        "Transmitted to Governor",
-        "Transmitted to House",
-        "Transmitted to House for Consideration of Governor's Proposed Amendments",
-        "Transmitted to Senate",
-        "Transmitted to Senate for Consideration of Governor's Proposed Amendments",
-        "Veto Overridden in House",
-        "Veto Override Failed in Legislature",
-        "Veto Override Motion Failed in House",
-        "Veto Override Motion Failed in Senate",
-        "Veto Override Vote Mail Poll Letter Being Prepared",
-        "Veto Override Vote Mail Poll in Progress",
-        "Vetoed by Governor",
-    ]
-)
 
+# TODO: write function description comment, remove unncessary comments, CHANGE MT INIT FILE BACK
```

Comment:
Looks like you've done this so can be removed?

Reply:
thanks for catching that!